chapters/functions.tex

```diff
@@ -439,17 +444,15 @@ \subsection{Positional or Named Input Arguments of Functions}\doublelabel{positi
 order of the parameters is given by the order of the component
 declarations in the function definition. Next, for each named argument
 \lstinline!identifier = expression!, the \lstinline!identifier! is used to determine the
-corresponding slot. This slot shall be not filled {[}\emph{otherwise an
-error occurs}{]} and the value of the argument is placed in the slot,
+corresponding slot. It is an error if this slot is already filled, and the value of the argument is placed in the slot,
```

Comment:
This sentence looks a bit weird to me, since it starts with an error and then has the normal important behavior.
I would prefer something like "The value of the argument is placed in the slot (it is an error if this slot is already filled), ..."

Reply:
I put the parenthesis with the error condition after _filling it_, to avoid breaking up the sentence too much with the large parenthesis in the middle.